script.js

```diff
@@ -898,6 +898,7 @@
                         font-size: 14px;
                         padding: 16px;
                         line-height: 20px;
+                        border-radius: 20px;
```

Comment:
![image](https://user-images.githubusercontent.com/20955511/224541997-5ff3c5a9-529f-47fa-b785-4f22635ccec6.png)
![image](https://user-images.githubusercontent.com/20955511/224542085-2014e14f-b3e5-48bf-ba02-25a6fee51849.png)


```suggestion
                        border-radius: 4px;
```